packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsWithAuthTest.e2e.test.ts

```diff
@@ -589,6 +601,11 @@ beforeAll(async () => {
         apiKey,
       },
     });
+    API.configure({
+      aws_appsync_graphqlEndpoint: GRAPHQL_ENDPOINT,
+      aws_appsync_region: AWS_REGION,
+      aws_appsync_authenticationType: 'AMAZON_COGNITO_USER_POOLS',
```

Comment:
we can also add the api key here as well and then change the auth modes when executing the gql operations.